### ISSUE TITLE
:in modifier

### ISSUE THREAD

**Issue by reubenelder** [microsoft/fhir-server#4694]
Hi,

From what I understand the :in modifier is not supported.  For example:

GET {{fhirurl}}/Medication?code:in={valueseturl}

Is this on the roadmap or are there no plans for this modifier to be added to the server?

Thanks,
Reuben

**Comment by EXPEkesheth** [comment]
@reubenelder Yes :in modifier is on the roadmap. There is no ETA at this time. 
Can you please share your scenario with :in modifier?

**Comment by reubenelder** [comment]
thanks @EXPEkesheth - we are a drug formulary and we have medication resources for all medications in our country.  We are starting to provide valuesets with subsets of those medications for various purposes and it is much easier for our vendors if they can use the :in modifier to get the medications resources from the valueset if needed.  Our valueset(s) will all be Extensional if that helps (I imagine that is a lot simpler.

**Comment by EXPEkesheth** [comment]
Tagging the request with internal backlog item [85030](https://microsofthealth.visualstudio.com/Health/_workitems/edit/85030)

**Comment by reubenelder** [comment]
Hi @EXPEkesheth , just checking how this task is going - is it still on the backlog?

**Comment by evachen96** [comment]
Hey @reubenelder , yes, this is still in the backlog!

**Comment by reubenelder** [comment]
Hi, just checking in again around this functionality - is there an expected timeframe for implementation?

**Comment by evachen96** [comment]
Hi @reubenelder , this is still in the backlog.

**Comment by kashish2210** [comment]
i would like to work on this may i proceed?
